authutils/oauth2/client/authorize.py

```diff
@@ -24,5 +24,5 @@ def client_do_authorize():
         return token
     except KeyError as e:
         raise AuthError("error in token response: {}".format(token))
-    except (OAuth2Error, OAuthException) as e:
+    except (OAuth2Error, AuthlibBaseError) as e:
```

Comment:
if you handle these both in the same `except` block I think technically it doesn't matter whether you catch both or only `AuthlibBaseError`; with the former inheriting from the latter `except AuthlibBaseError` would catch everything